sdk/react/src/DVCProvider.tsx

```diff
@@ -11,17 +11,17 @@ type Props = {
 
 export default function DVCProvider(props: Props): React.ReactElement {
     const { envKey, user, options } = props.config
-    const [client, setClient] = useState<DVCClient | undefined>(undefined)
+    const [client, setClient] = useState<DVCClient>(undefined!)
```

Comment:
you can't default this state to undefined otherwise you're violating the type definition. It needs to be set to a valid client at all times

Reply:
Cool! Learned about `useMemo`